build.gradle

```diff
@@ -79,6 +89,7 @@ allprojects {
 
 subprojects { project ->
 
+    apply plugin: 'net.ltgt.apt'
```

Comment:
Not all subprojects request apt. Only `:api-client` (the other one's are android libraries.